examples/vault/encrypt_structured/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>cloud.pangeacyber.examples</groupId>
+  <artifactId>encrypt_structured</artifactId>
+  <version>1.0.0</version>
+
+  <name>vault_encrypt_structured</name>
+  <url>https://pangea.cloud</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.release>17</maven.compiler.release>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>ossrh</id>
+      <name>ossrh-snapshot</name>
+      <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
+      <snapshots>
+        <enabled>true</enabled>
+        <updatePolicy>always</updatePolicy>
+      </snapshots>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+    </repository>
+  </repositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>cloud.pangea</groupId>
+      <artifactId>pangea-sdk</artifactId>
+      <version>3.5.0</version>
```

Comment:
We should update this version before release it

Reply:
Alright: https://github.com/pangeacyber/pangea-java/pull/188

Reply:
Done.